backend/oasst_backend/api/v1/tasks.py

```diff
@@ -276,27 +277,45 @@ async def tasks_interaction(
                 )
 
                 # here we store the text reply in the database
-                newMessage = pr.store_text_reply(
+                new_message = pr.store_text_reply(
                     text=interaction.text,
                     frontend_message_id=interaction.message_id,
                     user_frontend_message_id=interaction.user_message_id,
                 )
 
+                if not settings.DEBUG_SKIP_TOXICITY_CALCULATION:
+                    try:
+                        model_name: str = HfClassificationModel.TOXIC_ROBERTA.value
+                        hugging_face_api: HuggingFaceAPI = HuggingFaceAPI(
+                            f"{HfUrl.HUGGINGFACE_TOXIC_ROBERTA.value}/{model_name}"
+                        )
+
+                        toxicity: List[List[ToxicityClassification]] = await hugging_face_api.post(interaction.text)
```

Comment:
`post` returns `Any`. casting like this is really suboptimal, and especially to pydantic objects.
have you tested whether all of this works? if yes, we can leave it like this and fix later, but given that http calls return json as dicts, and later in your code you act properties of toxicity, it seems quite dangerous.